### PR TITLE
Log To Metrics Filter Plugin Documentation

### PR DESCRIPTION
**Pull request by drbugfinder-work**
Documentation for the new https://github.com/fluent/fluent-bit/pull/6674 Log To Metrics Filter Plugin

### CODE REVIEW

File: pipeline/filters/log-metrics.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
+---
+description: Generate metrics from logs
+---
+
+# Log Metrics
+
+The _Log Metrics Filter_ plugin allows you to generate log-derived metrics. It currently supports modes to count records, sum up field values over a record stream or provide a gauge for field values. You can also match or exclude specific records based on regular expression patterns for values or nested values. This filter plugin does not actually act as a record filter and does not change or drop records.
```
Comment (PettitWesley) [inline pipeline/filters/log-metrics.md:7]: > This filter plugin does not actually act as a record filter and does not change or drop records.

What does it do then? Pass on the logs as-is and then create new records in the metrics pipeline? Please make this more clear.

**Comment by PettitWesley** [comment]
@drbugfinder-work See the style of commits on this repo: https://github.com/fluent/fluent-bit-docs/commits/master

Typically we have one commit per file changed and its prefixed with the component.

**Comment by drbugfinder-work** [comment]
> @drbugfinder-work See the style of commits on this repo: https://github.com/fluent/fluent-bit-docs/commits/master
> 
> 
> 
> Typically we have one commit per file changed and its prefixed with the component. 
Oh, I see. I will correct this.

**Comment by drbugfinder-work** [comment]
> @drbugfinder-work See the style of commits on this repo: https://github.com/fluent/fluent-bit-docs/commits/master
> 
> 
> 
> Typically we have one commit per file changed and its prefixed with the component. 



Oh, I see. I will correct this.

**Comment by drbugfinder-work** [comment]
> Typically we have one commit per file changed and its prefixed with the component.
@PettitWesley 
Done.

**Comment by drbugfinder-work** [comment]
Added note for experimental feature.

**Pull request merged**